botnim/config.py

```diff
@@ -14,10 +15,68 @@
 
 def get_logger(name: str) -> logging.Logger:
     """Get a logger instance for the given name"""
-    return logging.getLogger(name)
+    logger = logging.getLogger(name)
+    logger.setLevel(logging.INFO)
+    if not logger.handlers:
+        handler = logging.StreamHandler()
+        handler.setFormatter(logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s'))
+        logger.addHandler(handler)
+    return logger
 
 
 # Embedding model settings
 DEFAULT_EMBEDDING_MODEL = 'text-embedding-3-small'
 DEFAULT_EMBEDDING_SIZE = 1536
 DEFAULT_BATCH_SIZE = 50
+
+# Constants
+VALID_ENVIRONMENTS = ['production', 'staging']
+DEFAULT_ENVIRONMENT = 'staging'
+
+def validate_environment(environment: str) -> str:
+    """
+    Validate that the environment is one of the valid options.
+    
+    Args:
+        environment (str): The environment to validate
+        
+    Returns:
+        str: The validated environment
+        
+    Raises:
+        ValueError: If the environment is not valid
+    """
+    if environment not in VALID_ENVIRONMENTS:
+        raise ValueError(f"Invalid environment: {environment}. Must be one of: {', '.join(VALID_ENVIRONMENTS)}")
+    return environment
+
+def is_production(environment: str) -> bool:
+    """
+    Check if the environment is production.
+    
+    Args:
+        environment (str): The environment to check
+        
+    Returns:
+        bool: True if the environment is production, False otherwise
+    """
+    validate_environment(environment)
+    return environment == 'production'
+
+def get_index_name(bot_slug: str, context_name: str, environment: str) -> str:
```

Comment:
The global config file should _not_ contain any such backend-specific method...

Reply:
moved.